simple-stack-example-kotlin-dagger/src/main/java/com/zhuinden/simplestackkotlindaggerexample/userdetail/UsersAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.zhuinden.simplestackkotlindaggerexample.userdetail
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.zhuinden.simplestackkotlindaggerexample.MainActivity
+import com.zhuinden.simplestackkotlindaggerexample.R
+import com.zhuinden.simplestackkotlindaggerexample.realmobjects.UserRO
+import io.realm.RealmRecyclerViewAdapter
+import io.realm.RealmResults
+
+class UsersAdapter(
+    private val items: RealmResults<UserRO>?
+) : RealmRecyclerViewAdapter<UserRO, UsersViewHolder>(items, true) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): UsersViewHolder =
+        UsersViewHolder(LayoutInflater.from(parent.context)
+            .inflate(R.layout.view_item_contact, parent, false))
+
+    override fun onBindViewHolder(usersViewHolder: UsersViewHolder, position: Int) {
+        val item = items!![position]
```

Comment:
You should use `getData()` instead so that if RealmResults is replaced, then you access the new RealmResults provided to this adapter